cmd/storagenode-updater/main_test.go

```diff
@@ -76,8 +191,18 @@ func TestAutoUpdater(t *testing.T) {
 		Binary: versioncontrol.ProcessesConfig{
 			Storagenode: versioncontrol.ProcessConfig{
 				Suggested: versioncontrol.VersionConfig{
-					Version: "0.19.5",
-					URL:     ts.URL + "/download",
+					Version: newVersion,
+					URL:     ts.URL + "/storagenode",
+				},
+				Rollout: versioncontrol.RolloutConfig{
+					Seed:   "0000000000000000000000000000000000000000000000000000000000000001",
```

Comment:
we should have these seeds random - just to have more realistic testing.

Reply:
but with %100 cursor?

Reply:
The cursor should be fixed at 100% to ensure that the update will trigger

Reply:
A part of @mniewrzal said, if I don't remember wrong, there is a uni test for ensuring that the rollout works well with more realistic values.

Reply:
I will add TODO but I don't want to play with this now